dist/tools/travis-scripts/get-pkg-list.py

```diff
@@ -29,7 +29,7 @@
 known_mcu_groups = arm_mcu_groups + msp_mcu_groups + x86_mcu_groups + \
     avr8_mcu_groups + static_tests_groups
 
-common_pkgs = ["pcregrep", "libpcre3", "python3"]
+common_pkgs = ["pcregrep", "libpcre3", "python3", "python3-pexpect"]
```

Comment:
(this is important if we want to run multiple tests without starting a new tap interface for every test with native [because the unkilled process of a previous test is hogging its assigned TAP])